src/vaadin-text-field-mixin.html

```diff
@@ -680,10 +714,16 @@
       }
     }
 
-    _getActiveErrorId(invalid, errorMessage, errorId) {
-      this._setOrToggleAttribute('aria-describedby',
-        (errorMessage && invalid ? errorId : undefined),
-        this.focusElement);
+    _getActiveErrorId(invalid, errorMessage, errorId, helperText, helperTextId) {
+      let ids = '';
+      if (helperText) {
+        ids += helperTextId;
+      }
+      if (errorMessage && invalid) {
+        ids += helperText ? ' ' : '';
+        ids += errorId;
+      }
+      this._setOrToggleAttribute('aria-describedby', ids, this.focusElement);
```

Comment:
A slotted helper text doesn't seem to label the focusElement. Is this intentional (as slotted content might not be text)?

Reply:
Ah I totally missed this. I changed it to add the label on slotchange.

I think if slotted content has no text then maybe it is up to the dev to handle the a11y issues?

Tested here: [1](https://github.com/vaadin/vaadin-text-field/blob/4a20bf4aa9f84b881367d33740bf14a4bf7bc36e/test/accessibility.html#L327).

Reply:
Done